### PR TITLE
CromwellOnAzure + Azure SQL DB + AAD first steps doc [VS-805]

### PR DESCRIPTION
**Pull request by mcovarr**

### CODE REVIEW

**Review by gbggrant — APPROVED** [review]
LGTM

File: scripts/variantstore/docs/azure/cromwell_on_azure.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+SUBSCRIPTION_ID=$(az account list | jq -r '.[] | select(.name | test("variants")) | .id')
+
+dotnet src/deploy-cromwell-on-azure/bin/Debug/net6.0/deploy-cromwell-on-azure.dll --SubscriptionId ${SUBSCRIPTION_ID} --RegionName eastus --MainIdentifierPrefix ${USER}
+```
+
+The CoA deployment can take more than 20 minutes to complete. Once that's done:
+
+```
+# Fish out the CromwellOnAzure resource group name that was named after your Broad username.
+RESOURCE_GROUP=$(az group list | jq -r ".[] | .name | select(test(\"^${USER}-[0-9a-f]+$\"))")
```
Comment (RoriCremer) [inline scripts/variantstore/docs/azure/cromwell_on_azure.md:27]: maybe a link to documentation on what on earth a resource group is?

**Review by RoriCremer — APPROVED** [review]
my review is definitely colored by the fact that I do not know what a lot of these Azure things are and maybe my questions dont make sense _here_ but I still have them.
Note: I have not run this. Should I?

**Comment by mcovarr** [comment]
> Note: I have not run this. Should I?

While I don't think it's absolutely essential at this point I certainly wouldn't discourage it. 😄 Happy to walk through this at mobbing or some other time if you like.

**Pull request merged**